gtests/net/mptcp/mp_join/mp_join_server_v4.pkt

```diff
@@ -2,6 +2,7 @@
 `../common/defaults.sh`
 
 +0    `../common/server.sh`
++0    `nstat -r -n`
```

Comment:
maybe you didn't want to commit this?

Reply:
@matttbe you got me, I use to insert `nstat | grep MPTcpBlah` in my scripts to trigger test failures in case my message is malformed. Sure, I will remove it :-)